lib/linalg/BasisReader.cpp

```diff
@@ -135,7 +151,7 @@ BasisReader::getTemporalBasis()
     int num_cols = getNumSamples("temporal_basis");
 
     char tmp[100];
-    Matrix* temporal_basis_vectors = new Matrix(num_rows, num_cols, true);
+    Matrix* temporal_basis_vectors = new Matrix(num_rows, num_cols, false);
```

Comment:
Why is this changed to non-distributed? Was it a bug for it to be distributed in the past?

Reply:
I do think it was a bug in the past. When I'm looking at `ComputeSVD`, temporal basis matrix is clearly stored as a non-distributed matrix (each process owns the global matrix)